<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task2</class>
 <widget class="QDialog" name="Task2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Task2</string>
  </property>
  <widget class="QTextEdit" name="teipv4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teSubnetMask">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teBroadcastAddress">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ipv4 address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>subnet mask</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGetBA">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get broadcast address</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
